### PR TITLE
Feature landing

### PR DESCRIPTION
**Pull request by poudyalanil**
Tasks Done:

- Created separate components for landing page
- Created MVP of `account` page
- Create private route for `account`

### CODE REVIEW

**Comment by timilsinabishal** [comment]
@poudyalanil please squash all commits into a single commit

File: public/index.html [hunk 1]
```diff
@@ -24,12 +22,14 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+  <title>React App</title>
```
Comment (timilsinabishal) [inline public/index.html]: Change the title to "NEAT+ - Nexus Environment Assessment Tool"

File: public/index.html [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (timilsinabishal) [inline public/index.html]: lets replace the favicon

File: public/index.html [hunk 3]
```diff
@@ -39,5 +39,9 @@
...
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+
+
+
+</body>
+
+</html>
```
Comment (timilsinabishal) [inline public/index.html:45]: new line

File: src/components/About/style.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
+p {
```
Comment (timilsinabishal) [inline src/components/About/style.css]: use CSS modules for CSS

File: src/components/Accordin/index.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+const Accordin = ({ title, content }) => {
+    const [isActive, setIsActive] = useState(false);
+
+    return (
+        <div className={styles.accordin}>
+            <div className={styles.accordin_trigger} onClick={() => setIsActive(!isActive)}>
+                <div className={styles.accordin_title}>
+
+
+                    <p style={{ marginRight: '5px' }}>{title}
```
Comment (timilsinabishal) [inline src/components/Accordin/index.js]: no inlines  (not anywhere)

File: src/components/Accordin/index.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+                    </p>
+
+                    <p>
+
+                        {isActive ? <MdClose style={{ color: '#737373', backgroundColor: '#f2f2f2', borderRadius: '50%', padding: '5px' }} size="30px" /> : <BsArrowRight size="30px" />}
```
Comment (timilsinabishal) [inline src/components/Accordin/index.js]: same no inlines

File: src/components/Accordin/style.css [hunk 7]
```diff
@@ -0,0 +1,19 @@
+.accordin {
+  background-color: white;
+  margin-bottom: 10px;
+  border-radius: 10px;
+  padding: 10px;
+}
+.accordin_trigger {
```
Comment (timilsinabishal) [inline src/components/Accordin/style.css]: typo "accordin"

File: src/components/Accordin/style.css [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+  background-color: white;
+  color: #00a297;
+  padding: 10px;
+  cursor: pointer;
+}
+.accordin_text {
```
Comment (timilsinabishal) [inline src/components/Accordin/style.css]: typo

File: src/components/Accordin/style.css [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+  color: #000;
+}
+.accordin_title {
```
Comment (timilsinabishal) [inline src/components/Accordin/style.css]: typo

File: src/components/AppInfo/index.js [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+                <center className={styles.info_note}>
+                    <small>* a multi-stakeholder project aimed at improving collaboration between environmental and humanitarian actors</small>
+                </center>
+
+            </section>
+
+            <section className={styles.feedback}>
+                <div className={styles.feedback_text}>
+
+                    <h1 style={{ color: '#fff' }}>
```
Comment (timilsinabishal) [inline src/components/AppInfo/index.js]: no inlines!!!

File: src/components/AppInfo/index.js [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+                        To continuously improve the <strong> NEAT+</strong> and adapt it to user’s needs, we would like to learn
+                more about your
+                experience as a
+                user and the context in which you used the tool in. Take 10 minutes to fill out this  <a href="#"> <strong>NEAT+ user feedback form </strong> </a>. All
+                responses are confidential and anonymous.
+                    </p>
+                    <p className={styles.feedback_para}>
+                        If you have any further questions, comment or feedback, please contact the OCHA/UNEP Joint Environment Unit
+                        (JEU)
+                        <a style={{ cursor: 'pointer', color: '#00a297' }} href="mailto:ochaunep@un.org">
```
Comment (timilsinabishal) [inline src/components/AppInfo/index.js]: no inlines!!

File: src/components/AppInfo/index.js [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+                            (ochaunep@un.org)</a>.
+                    </p>
+                </div>
+
+                <div className={styles.feedback_visual}>
+                    <img src="https://res.cloudinary.com/anilpoudyal/image/upload/v1620986633/Group_661_goykdj.png" alt="" />
```
Comment (timilsinabishal) [inline src/components/AppInfo/index.js]: ??

**Comment by poudyalanil** [comment]
> @poudyalanil please squash all commits into a single commit

Sure

File: src/components/AppInfo/style.css [hunk 13]
```diff
@@ -0,0 +1,103 @@
+.neat_application {
```
Comment (timilsinabishal) [inline src/components/AppInfo/style.css]: use CSS modules (please refer mentors codebase)

also use hierarchical structure

File: src/components/AppInfo/trigger.js [hunk 14]
```diff
@@ -0,0 +1,14 @@
+
+let trigger = document.getElementsByClassName('accordin_trigger');
```
Comment (timilsinabishal) [inline src/components/AppInfo/trigger.js]: move logic to a component could be lot simpler and cleaner

File: src/components/AppInfo/trigger.js [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+let i;
+for (i = 0; i < trigger.length; i++) {
```
Comment (timilsinabishal) [inline src/components/AppInfo/trigger.js]: for ... in

File: src/components/Feature/index.js [hunk 16]
```diff
@@ -0,0 +1,95 @@
...
+                        associated with specific activities.
+                    </p>
+                </div>
+                <div></div>
+            </div>
+            <div className={styles.neat_feature_cards}>
+                <div className={styles.feature}>
+                    <img src="https://res.cloudinary.com/anilpoudyal/image/upload/v1620984783/savannah_qltphg.png" />
+
+                    <div style={{ marginLeft: '35px' }}>
```
Comment (timilsinabishal) [inline src/components/Feature/index.js]: inline

File: src/components/Feature/index.js [hunk 17]
```diff
@@ -0,0 +1,95 @@
...
+                            organizations), and produces the report in Excel. Alternatively, data can be input into Excel
+                            directly.
+                        </p>
+                    </div>
+
+                </div>
+
+                <div className={styles.feature}>
+                    <img src="https://res.cloudinary.com/anilpoudyal/image/upload/v1620984776/city_e0gvur.png" />
+                    <div style={{ marginLeft: '35px' }}>
```
Comment (timilsinabishal) [inline src/components/Feature/index.js]: inline

File: src/components/Feature/index.js [hunk 18]
```diff
@@ -0,0 +1,95 @@
...
+                        environmental assessment should be carried out. The tool produces results in an easy to read and
+                        understand format and
+                        does not require expertise in environmental topics. The suggested mitigations can be prioritised to
+                        strengthen projects
+                        against environmental vulnerabilities and reduce harmful project impacts on the environment.
+
+                    </p>
+                    <br />
+
+                    <p style={{ fontSize: '16px', lineHeight: '24px' }}>
```
Comment (timilsinabishal) [inline src/components/Feature/index.js]: inline??

File: src/components/Feature/style.css [hunk 19]
```diff
@@ -0,0 +1,59 @@
+h1 {
```
Comment (timilsinabishal) [inline src/components/Feature/style.css]: CSS modules

File: src/components/Footer/index.js [hunk 20]
```diff
@@ -0,0 +1,119 @@
...
+                <div className={styles.policy_links}>
+                    <a hreg="#">Privacy Policy</a>
+                    <a hreg="#">Terms of use</a>
+                    <a hreg="#">Cookie Policy</a>
+                </div>
+
+
+            </div>
+        </footer>
+
```
Comment (timilsinabishal) [inline src/components/Footer/index.js]: only single newline if used

File: src/components/Footer/style.css [hunk 21]
```diff
@@ -0,0 +1,63 @@
+/* footer section */
+footer {
```
Comment (timilsinabishal) [inline src/components/Footer/style.css]: CSS modules

File: src/components/Header/style.css [hunk 22]
```diff
@@ -0,0 +1,90 @@
+header > li,
```
Comment (timilsinabishal) [inline src/components/Header/style.css]: CSS modules and always use classnames

File: src/containers/Account/index.js [hunk 23]
```diff
@@ -0,0 +1,87 @@
+import UserNav from 'components/UserNav';
+import styles from './style.css';
+import Form from 'components/Form';
```
Comment (timilsinabishal) [inline src/containers/Account/index.js]: arrange imports

first core libraries
second npm libraries
third react-arsenal
four other custom components

File: src/containers/Account/style.css [hunk 24]
```diff
@@ -0,0 +1,126 @@
+body {
```
Comment (timilsinabishal) [inline src/containers/Account/style.css]: CSS modules

File: src/containers/Home/index.js [hunk 25]
```diff
@@ -1,18 +1,20 @@
-import {Link} from 'react-router-dom';
 
-import styles from './styles.scss';
+// import styles from './styles.scss';
```
Comment (timilsinabishal) [inline src/containers/Home/index.js]: no comments

File: src/routes/index.js [hunk 26]
```diff
@@ -1,13 +1,14 @@
-import {Route, Switch} from 'react-router-dom';
+import { Route, Switch } from 'react-router-dom';
```
Comment (timilsinabishal) [inline src/routes/index.js:1]: setup linting

File: src/containers/Home/About/styles.scss [hunk 27]
```diff
@@ -0,0 +1,65 @@
+.about-section {
+  padding: 120px 130px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  padding-top: 50px;
+  padding-bottom: 50px;
+  background-color: var(--color-white);
+  padding-top: 80px;
```
Comment (ft9dipesh) [inline src/containers/Home/About/styles.scss]: Too many padding properties. Just use one.

File: src/containers/Home/AppInfo/styles.scss [hunk 28]
```diff
@@ -0,0 +1,123 @@
...
+  .info-note {
+    padding-top: 100px;
+    font-size: var(--font-size-small);
+  }
+
+  .info-note {
+    margin-top: 20px;
+    font-size: var(--font-size-small);
+    line-height: 20px;
+  }
```
Comment (ft9dipesh) [inline src/containers/Home/AppInfo/styles.scss]: These are the same selectors. Use one.

File: src/containers/Home/Header/styles.scss [hunk 29]
```diff
@@ -0,0 +1,94 @@
...
+        border-style: none;
+        cursor: pointer;
+      }
+      .header-title {
+        font-size: 46px;
+        line-height: 62px;
+        margin-bottom: 20px;
+      }
+      .hero-text {
+        font: size 16px;
```
Comment (ft9dipesh) [inline src/containers/Home/Header/styles.scss]: ?!

**Pull request closed without merging**